docs/index.md

```
@@ -1,10 +1,82 @@
+![Cliffy logo](images/logo.svg)
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)